common/sgx/endorsements.c

```diff
@@ -22,48 +22,72 @@
  * @param[out] endorsements OE endorsement structure.
  */
 static oe_result_t oe_create_sgx_endorsements(
-    const oe_get_revocation_info_args_t* revocation_info,
-    const oe_get_qe_identity_info_args_t* qe_id_info,
+    const oe_get_quote_verification_collateral_args_t*
+        quote_verification_collateral,
     oe_endorsements_t** endorsements_buffer,
     size_t* endorsements_buffer_size)
 {
     oe_result_t result = OE_UNEXPECTED;
     oe_endorsements_t* endorsements = NULL;
+    // oe_cert_t root_ca_cert = {0};
+    // oe_cert_chain_t* pck_crl_issuer_chain = {0};
```

Comment:
Recommend either removing commented out code, or un-commenting it out.

Reply:
Thanks, removing it.